internal/discord/command-handlers.go

```diff
@@ -15,6 +15,14 @@ var (
 	adminPermission int64 = discordgo.PermissionAdministrator
 )
 
+// DiscordReady initializes the bot and blocks the bot from proceeding until
+// initialization finishes
+func (d *Discord) DiscordReady(s *discordgo.Session, event *discordgo.Ready) {
+	defer d.Ready.Done()
+	d.AddCommands(s, event)
+	d.MapExistingRoles(s, event)
+}
+
```

Comment:
it feels like this belongs more in `discord.go` instead of `command-handlers.go`